trento/migration/systemd-install.md

```diff
@@ -487,12 +435,78 @@ This is a basic guide for creating a self-signed certificate for use with Trento
 
    If there are issues with the configuration, the output will indicate what needs to be adjusted.
 
-1. Reload Nginx to apply changes:
+1. If SSL certificates were modified or added, reload NGINX to apply changes.
 
    ```bash
    systemctl reload nginx
    ```
 
+## Prepare SSL certificate for NGINX
+
+Create or provide a certificate for [NGINX](https://nginx.org/en/) to enable SSL for Trento.
+This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the [OpenSSL documentation](https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html).
+
+### Option 1: Creating a Self-Signed Certificate
+
+1.  Generate a self signed certificate:
+
+    > Note: Remember to adjust `subjectAltName = DNS:trento.example.com` by replacing `trento.example.com` with your own domain and change the value `5` to the number of days for which you need the certificate to be valid. For example, `-days 365` for one year.
```

Comment:
```suggestion
    > Note: Remember to adjust `subjectAltName = DNS:trento.example.com` by replacing `trento.example.com` with your the actual domain name, and change the value `5` to the number of days for the certificate validity. For example, `-days 365` for one year.
```

Reply:
in "with your the actual domain name", "the" should be removed, right?